src/app/app.module.ts

```diff
@@ -48,6 +49,7 @@ if (process.env.ENV=='inmemory')
     WorkItemDetailComponent,
     WorkItemQuickAddComponent,
     WorkItemListComponent,
+    WorkItemListEntryComponent,
```

Comment:
@michaelkleinhenz I guess this should go into WorkItemListComponent instead of main module. Because it's a sub component of WorkItemListComponent. If we keep adding all the sub components here, this list will be crazy big.